### ISSUE TITLE
Querying field's required value returns false

### ISSUE THREAD

**Issue by K-JAX** [toriphes/wp-graphql-ninja-forms#1]
I'm trying to query the form fields' required value but it returns false for all of them even after explicitly setting field's required value to true in the Ninja Form editor. Here's an example query I'm using:
```graphql
query MyQuery {
  __typename
  form(id: "1", idType: DATABASE_ID) {
    title
    fields {
      nodes {
        label
        required
      }
    }
  }
}
```

And this outputs:
```graphql
{
  "data": {
    "__typename": "RootQuery",
    "form": {
      "title": "Contact Me",
      "fields": {
        "nodes": [
          {
            "label": "Name",
            "required": false
          },
          {
            "label": "Email",
            "required": false
          },
          {
            "label": "Message",
            "required": false
          },
          {
            "label": "Submit",
            "required": null
          }
        ]
      }
    }
  }
}
```
..^returning three false values for each required field.

Any ideas?

**Comment by toriphes** [comment]
Hi @K-JAX thanks for the feedback.
I just released version 0.1.1 which fixes this bug.

Please let me know if everything works fine.

**Comment by K-JAX** [comment]
@toriphes just ran a query with the updated version. Looks good! 
Thanks for the support pal! And thanks for this plugin!
